modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -527,6 +529,51 @@ def validate_reserves_tos_dates!
           )
         end
       end
+
+      def validate_form_526_direct_deposit!
+        direct_deposit = form_attributes['directDeposit']
+        return if direct_deposit.blank?
+
+        direct_deposit['noAccount'] == true ? validate_no_account! : validate_account_values!
+      end
```

Comment:
Question: What if there is not a value for `noAccount`? 

```
irb(main):009:0> form_attributes = { "directDeposit"=> { "accountType"=> 'Checking', 'accountNumber'=> '3453425235' } }
=> {"directDeposit"=>{"accountType"=>"Checking", "accountNumber"=>"3453425235"}}
irb(main):010:0> direct_deposit = form_attributes['directDeposit']
=> {"accountType"=>"Checking", "accountNumber"=>"3453425235"}
irb(main):011:0> direct_deposit['noAccount'] == true ? 'yes' : 'no'
=> "no"
```

Reply:
I think that is okay, just want to run it by you.

Reply:
you're right, it can be much more robust, updating that now